test/jdk/javax/swing/JButton/4796987/bug4796987.java

```diff
@@ -95,18 +118,27 @@ private static JButton getButton() {
     }
 
     private static void createAndShowGUI() {
-        frame = new JFrame("Test");
+        frame = new JFrame("bug4796987");
         frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
         frame.setSize(200, 200);
 
         JButton button = new JButton();
         button.setBorder(null);
```

Comment:
What's the use of this `button` instance ? Looks unused instance to me and can be removed.